src/Setup/Setup.csproj

```diff
@@ -21,7 +21,7 @@
     </ItemGroup>
     <PropertyGroup>
       <SetupExeOutputFolder>$(SolutionDir)..\assets\</SetupExeOutputFolder>
-      <SetupExeName>Particular.ServicePulse-$(GitVersion_MajorMinorPatch).exe</SetupExeName>
+      <SetupExeName>Particular.ServicePulse-$(MinVerMajor).$(MinVerMinor).$(MinVerPatch.exe</SetupExeName>
```

Comment:
This should probably have been `<SetupExeName>Particular.ServicePulse-$(MinVerMajor).$(MinVerMinor).$(MinVerPatch).exe</SetupExeName>`

Reply:
Yes, that is a typo.